tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -205,14 +205,20 @@ def test_it_reports_lti_launches(
 
     @pytest.mark.usefixtures("user_is_learner")
     def test_it_calls_grading_info_upsert(
-        self, context, pyramid_request, grading_info_service, view_caller
+        self,
+        context,
+        pyramid_request,
+        grading_info_service,
+        view_caller,
+        application_instance,
     ):
         view_caller(context, pyramid_request)
 
         grading_info_service.upsert_from_request.assert_called_once_with(
             pyramid_request,
-            h_user=pyramid_request.lti_user.h_user,
-            lti_user=pyramid_request.lti_user,
+            pyramid_request.lti_user.h_user,
+            pyramid_request.lti_user.user_id,
+            application_instance,
         )
```

Comment:
A little more fallout from the extra arguments. This also kind of demonstrates the point that this all comes from the request object too.